docker-compose.yml

```diff
@@ -24,27 +24,23 @@ services:
         test: ["CMD", "pg_isready", "-U", "${POSTGRES_USER}", "-d", "${POSTGRES_DB}"]
   web:
     container_name: web
-    build:
-      context: .
-      dockerfile: Dockerfile
+    image: codeforafrica/ckan:${CKAN_VERSION}
```

Comment:
Minor but does this mean we will no longer be able to quickly build and test locally before bumping version or making any customisations?